FhirToDataLake/deploy/templates/DeployAzureContainerApp.json

```diff
@@ -39,28 +39,46 @@
             "description": "The name of the container instance that you wish to create."
         }
     },
-    "fhirServerUrl": {
+    "dataSourceType": {
         "type": "string",
+        "defaultValue": "FHIR",
+        "allowedValues": ["FHIR", "DICOM"],
         "metadata": {
-            "description": "The fhir server endpoint you wish to get data."
+            "description": "The source API server type you wish to get data."
         }
     },
-    "authentication": {
-        "type": "bool",
-        "defaultValue": false,
-        "allowedValues": [
-        true,
-        false
-        ],
+    "fhirServerUrl": {
+        "type": "string",
+        "defaultValue": "",
         "metadata": {
-            "description": "Whether to access Azure API for FHIR with managed identity authentication. Set false if you are using an OSS public FHIR server instance."
+            "description": "The fhir server endpoint you wish to get data."
         }
     },
     "fhirVersion": {
         "type": "string",
         "defaultValue": "R4",
         "allowedValues": ["R4"]
     },
+    "dicomServerUrl": {
+        "type": "string",
+        "defaultValue": "",
+        "metadata": {
+            "description": "The dicom server endpoint you wish to get data."
+        }
+    },
+    "dicomApiVersion": {
+        "type": "string",
+        "defaultValue": "V1",
+        "allowedValues": ["V1"]
```

Comment:
Do we allow "V1_0_Prerelease" here?

Reply:
Pre offline discussion, we firstly support V1 and after confirming the pipeline also work for "V1_0_Prerelease" we can add the "V1_0_Prerelease" option here.